protos/dcs/mission/v0/mission.proto

```diff
@@ -434,12 +455,45 @@ message StreamUnitsResponse {
 
 message GetScenarioStartTimeRequest {}
 
-message GetScenarioStartTimeResponse {
-  string datetime = 1;
-}
+message GetScenarioStartTimeResponse { string datetime = 1; }
 
 message GetScenarioCurrentTimeRequest {}
 
-message GetScenarioCurrentTimeResponse {
-  string datetime = 1;
+message GetScenarioCurrentTimeResponse { string datetime = 1; }
+
+message AddMissionCommandRequest {
+  string id = 1;
+  string name = 2;
+  repeated string path = 3;
+
+  oneof target {
+    bool global = 4;
+    dcs.common.v0.Coalition coalition = 5;
+    string group = 6;
+  }
+}
+
+message AddMissionCommandResponse {}
+
+message RemoveMissionCommandItemRequest {
+  repeated string path = 1;
+  oneof target {
+    bool global = 2;
+    dcs.common.v0.Coalition coalition = 3;
+    string group = 4;
+  }
 }
+
+message RemoveMissionCommandItemResponse {}
+
+message AddMissionCommandSubMenuRequest {
+  string name = 1;
+  repeated string path = 2;
+  oneof target {
+    bool global = 3;
+    dcs.common.v0.Coalition coalition = 4;
+    string group = 5;
```

Comment:
```suggestion
    string groupName = 5;
```
I've seen a few API's refer to it as `groupId` or `groupName`.

Yeah, I agree that we should try to stick to `groupId` as much as possible.

Reply:
I'm changing my suggestion to `groupName` - see the motivation as described here: https://github.com/DCS-gRPC/rust-server/pull/11#issuecomment-903350183